rpc/common/service.pb.go

```diff
@@ -366,7 +366,7 @@ type Vulnerability struct {
 	Description          string   `protobuf:"bytes,6,opt,name=description,proto3" json:"description,omitempty"`
 	Severity             Severity `protobuf:"varint,7,opt,name=severity,proto3,enum=trivy.common.Severity" json:"severity,omitempty"`
 	References           []string `protobuf:"bytes,8,rep,name=references,proto3" json:"references,omitempty"`
-	LayerId              string   `protobuf:"bytes,9,opt,name=layer_id,json=layerId,proto3" json:"layer_id,omitempty"`
+	Layer                *Layer   `protobuf:"bytes,10,opt,name=layer,proto3" json:"layer,omitempty"`
```

Comment:
I'm just curious: Why is this by reference? Are custom types always passed by reference?

Reply:
I'm not sure, but other custom types are also references.
https://github.com/aquasecurity/trivy/blob/1b112c30ca48cb2fbf5750f428665eea624d4404/rpc/cache/service.pb.go#L32